pkg/oci/remote.go

```diff
@@ -28,6 +74,9 @@ func getRepository(reference string) (*remote.Repository, error) {
 			password := os.Getenv(registry + "_PASSWORD")
 
 			if hostname != "" && username != "" && password != "" {
+				if repos[hostname].Username != "" {
```

Comment:
You want `if _, ok := repos[hostname]; !ok {`

Reply:
I am doing `ok` and not `!ok`.

I want the log message to show if the value exists (was populated from the file) so the user knows that we are about to overwrite it.

Reply:
Whoops! Yes. That is what you want :)